src/cws_pattern.js

```diff
@@ -101,14 +106,18 @@ function get_crx_url(extensionID_or_url) {
     if (match) {
         return 'https://' + match[1] + '/firefox/downloads/file/' + match[2] + (match[3] || '/addon.xpi');
     }
+    match = mea_pattern.exec(url) || mea_download_pattern.exec(url);
+    if (match) {
+        return 'https://edge.microsoft.com/extensionwebstorebase/v1/crx?response=redirect&x=id%3D' + match[1] + '%26installsource%3Dondemand%26uc';
+    }
     // Chrome Web Store
     match = get_extensionID(extensionID_or_url);
     var extensionID = match ? match : extensionID_or_url;
 
     if (!/^[a-z]{32}$/.test(extensionID)) {
         return extensionID_or_url;
     }
-
+    
```

Comment:
Please remove this space change, squash all commits together with a meaningful description and I'll merge it.